src/main/java/com/synopsys/integration/alert/channel/jira/cloud/descriptor/JiraDescriptor.java

```diff
@@ -30,17 +30,20 @@
 
 @Component
 public class JiraDescriptor extends ChannelDescriptor {
-    public static final String KEY_JIRA_URL = "jira.cloud.url";
-    public static final String KEY_JIRA_ADMIN_EMAIL_ADDRESS = "jira.cloud.admin.email.address";
-    public static final String KEY_JIRA_ADMIN_API_TOKEN = "jira.cloud.admin.api.token";
-    public static final String KEY_JIRA_CONFIGURE_PLUGIN = "jira.cloud.configure.plugin";
-
-    public static final String KEY_ADD_COMMENTS = "channel.jira.cloud.add.comments";
-    public static final String KEY_ISSUE_CREATOR = "channel.jira.cloud.issue.creator";
-    public static final String KEY_JIRA_PROJECT_NAME = "channel.jira.cloud.project.name";
-    public static final String KEY_ISSUE_TYPE = "channel.jira.cloud.issue.type";
-    public static final String KEY_RESOLVE_WORKFLOW_TRANSITION = "channel.jira.cloud.resolve.workflow";
-    public static final String KEY_OPEN_WORKFLOW_TRANSITION = "channel.jira.cloud.reopen.workflow";
+    public static final String JIRA_CLOUD_PREFIX = "jira.cloud";
```

Comment:
It may be easier to read if this prefix ended with a period and all other variables removed the starting period

Reply:
This recommendation can be applied to all changes in this PR